recipe/meta.yaml

```diff
@@ -1,63 +1,126 @@
-{% set name = "pennylane-lightning" %}
-{% set version = "0.24.0" %}
+{% set name = "pennylane" %}
+{% set version = "0.29.0" %}
 
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/PennyLaneAI/pennylane-lightning/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 38bfabb2f3984461652bff4e931469bf623befdcfeb888803e6685a852e6fa60
-  patches:
-    - 0001-Non-ninja.patch
-    # Ninja is not recognised by pip check.
+  - git_url: https://github.com/PennyLaneAI/pennylane.git
+    git_tag: v{{ version }}
+    folder: pennylane
+  - git_url: https://github.com/PennyLaneAI/pennylane-lightning.git
+    git_tag: v{{ version }}
+    folder: pennylane-lightning
 
 build:
   number: 0
-  # skip: true  # [win and vc<14]
   skip: true  # [win]
-  # Kokkos is currently not available
-  script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - setuptools
     # Needed for Windows
     - cmake
     - ninja
     - make
   host:
-    - kokkos
-    - libblas
-    - liblapacke
-    - libgomp      # [linux]
-    - llvm-openmp  # [osx]
-    - pip
-    - pybind11
     - python
   run:
-    - ninja
-    - numpy
-    - pennylane >=0.19
-    - python
-
+    - pennylane-core
+    - pennylane-lightning-core
+    
 test:
   imports:
-    - pennylane_lightning
+    - pennylane
   commands:
+    - python -c "import pennylane as qml; qml.about()"
     - pip check
+    - pl-device-test --help
+    - pl-device-test --device default.qubit --shots None --skip-ops
+    - pl-device-test --device default.qubit --shots 10000 --skip-ops
+    - pl-device-test --device lightning.qubit --shots None --skip-ops
+    - pl-device-test --device lightning.qubit --shots 10000 --skip-ops
+
   requires:
+    - setuptools <=64.0.2 # version requirement from cvxpy
+    - flaky
     - pip
+    - pytest
+    - pytest-mock
+
+outputs:
+
+  - name: pennylane-core
+    version: {{ version }}
+    build:
+      run_exports:
+        - {{ pin_subpackage('pennylane-core', max_pin='x.x') }}
+      entry_points:
+        - pl-device-test=pennylane.devices.tests:cli
+      script: cd "${SRC_DIR}/pennylane" && {{ PYTHON }} -m pip install . -vv
+      number: 0
+    requirements:
+      host:
+        - pip
+        - python
+      run:
+        - appdirs
+        - autograd
+        - autoray ==0.3.1
+        - cachetools
+        - cvxopt
+        - cvxpy
+        - networkx
+        - numpy <1.24
+        - opt_einsum
+        - python
+        - requests
+        - retworkx
+        - scipy
+        - semantic_version >=2.10
+        - toml
+
+  - name: pennylane-lightning-core
```

Comment:
This has now `-core`attached to it. Is that correct? Does the non-core version also exist or not anymore? (Just need that for the outputs)